crates/viewer/re_view_spatial/src/eye.rs

```diff
@@ -411,6 +419,19 @@ impl ViewEye {
             }
         });
 
+        let kind = eye_property.component_or_fallback::<Eye3DKind>(
+            view_ctx,
+            self,
+            &Eye3D::descriptor_kind(),
+        );
+        match kind {
```

Comment:
I suppose we will "merge" the enums `EyeMode` and `Eye3DKind` (either replace or rename) and then this `match` will go away.